src/Avalonia.Input/TextInput/InputMethodManager.cs

```diff
@@ -91,12 +92,15 @@ public void SetFocusedElement(IInputElement? element)
             _focusedElement = element;
 
             var inputMethod = (element?.VisualRoot as ITextInputMethodRoot)?.InputMethod;
-            if (_im != inputMethod)
-                _im?.SetActive(false);
-
+            
             _im = inputMethod;
 
             TryFindAndApplyClient();
+
+            if (_im != inputMethod)
```

Comment:
This logic already exists in Client property setter, I think

Reply:
removed.